nebula2/gclient/net/Connection.py

```diff
@@ -92,7 +95,10 @@ def _reopen(self):
         :return:
         """
         self.close()
-        self.open(self._ip, self._port, self._timeout)
+        if self._ssl_conf != None:
```

Comment:
is not None or if not self._ssl_conf

Reply:
Thanks for the correction. Seems I messed up with cpp syntax.